crates/polars-core/src/chunked_array/ops/apply.rs

```diff
@@ -115,6 +115,55 @@ where
 
         ChunkedArray::try_from_chunk_iter(self.name(), iter)
     }
+
+    pub fn apply_to_buffer_generic<'a, F>(&'a self, mut f: F) -> StringChunked
```

Comment:
Except, it shouldn't be called this, as it is not generic. It returns a `StringChunked`/

Maybe `apply_to_string_amortized`?

Reply:
true, I was just thinking that, respect to `apply_to_buffer`, the input can be any `PolarsDataType`

I think `apply_to_string_amortized` makes it sounds like the input is a `String`?

Reply:
Would `apply_into_string` be an OK name? `apply_to_string` to me just reads like the string is the input rather than the output